src/main/kotlin/no/nav/k9/domene/repository/StatistikkRepository.kt

```diff
@@ -373,4 +374,58 @@ class StatistikkRepository(
             )
         }
     }
+
+    fun hentReservasjoner(): List<Reservasjon> {
```

Comment:
StatistikkRepository er laget litt for å slippe å måtte forholde seg til Reservasjon, Oppgave osv, men bare gjøre spørringen direkte mot db og hente ut kun data som trengs for den enkelte statistikkoppgaven.

Reply:
Godt poeng. Som Qadeer nevnte ☝🏻 så skriver jeg nok om til å gjøre aggregeringen i form av en spørring. Så trenger ikke StatistikkRepository ha noe forhold til domeneobjektene